app/src/main/java/com/pokerio/app/components/PlayerView.kt

```diff
@@ -2,58 +2,158 @@ package com.pokerio.app.components
 
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.width
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.OutlinedCard
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.pokerio.app.R
+import com.pokerio.app.utils.GameState
 import com.pokerio.app.utils.Player
 import com.pokerio.app.utils.PlayerProvider
 
+val TEXT_MODIFIER = Modifier.padding(2.dp)
+val AMOUNT_SIZE = 11.sp
+val LABEL_SIZE = 10.sp
+
+// val NICK_BACKGROUND_COLOR = Color.
+val PLAYER_VIEW_CARD_SHAPE =
```

Comment:
This is a much simpler and also valid constructor:

`val PLAYER_VIEW_CARD_SHAPE = RoundedCornerShape(8.dp)`

Reply:
Done.